spinalcordtoolbox/scripts/sct_maths.py

```diff
@@ -434,59 +374,190 @@ def main(argv: Sequence[str]):
+                    # Averaging over a spatial axis, we should preserve it
+                    data = np.expand_dims(data, axis)
+
+        elif arg_name == "rms":
+            data = data.astype(float)
+            axis = ('x', 'y', 'z', 't').index(arg_value)
+            if axis >= data.ndim:
+                # Taking the mean across time for a 3D image has no effect.
+                # Because of this, RMS is just squaring then sqrting (i.e. abs)
+                data = np.abs(data)
+            else:
+                data = np.sqrt(np.mean(np.square(data), axis))
+                if axis < 3:
+                    # Taking RMS over a spatial axis, we should preserve it
+                    data = np.expand_dims(data, axis)
+
+        elif arg_name == "std":
+            axis = ('x', 'y', 'z', 't').index(arg_value)
+            if axis >= data.ndim or data.shape[axis] == 1:
+                printv("ERROR: Zero division while taking -std along a singleton dimension", 1, 'error')
+            else:
+                data = np.std(data, axis, ddof=1),
+                if axis < 3:
+                    # Taking std over a spatial axis, we should preserve it
+                    data = np.expand_dims(data, axis)
+
+        elif arg_name == "bin":
+            bin_thr = arg_value
+            data = sct_math.binarize(data, bin_thr)
+
+        elif arg_name == "otsu":
+            nbins = arg_value
+            data = sct_math.otsu(data, nbins)
+
+        elif arg_name == "adap":
+            block_size, offset = arg_value
+            data = sct_math.adap(data, block_size, offset)
+
+        elif arg_name == "otsu_median":
+            size, n_iter = arg_value
+            data = sct_math.otsu_median(data, size, n_iter)
+
+        elif arg_name == "percent":
+            percentile = arg_value
+            data = sct_math.perc(data, percentile)
+
+        elif arg_name == "thr":
+            threshold = arg_value
+            data = sct_math.threshold(data, lthr=threshold)
+
+        elif arg_name == "uthr":
+            threshold = arg_value
+            data = sct_math.threshold(data, uthr=threshold)
+
+        elif arg_name == "dilate":
+            # This uses the global `shape` and `dim` values
+            size = arg_value
+            data = sct_math.dilate(data, size, shape, dim)
+
+        elif arg_name == "erode":
+            # This uses the global `shape` and `dim` values
+            size = arg_value
+            data = sct_math.erode(data, size, shape, dim)
+
+        elif arg_name in ["smooth", "laplacian"]:
+            # Adjust sigmas from millimeters to voxels
+            # This uses the resolution of the starting value `im`
+            sigmas = [mm/pixdim for mm, pixdim in zip(arg_value, im.dim[4:7])]
+            if arg_name == "smooth":
+                data = sct_math.smooth(data, sigmas)
+            else:
+                assert arg_name == "laplacian"
+                data = sct_math.laplacian(data, sigmas)
```

Comment:
Let's use a dict here:
```suggestion
            data = {
                "smooth": sct_math.smooth,
                "laplacian": sct_math.laplacian,
            }[arg_name](data, sigmas)
```